### ISSUE TITLE
make test-integration fails on a clean docker daemon

### ISSUE THREAD

**Issue by arschles** [kubernetes-incubator/service-catalog#493]
I launched a brand new VM running minikube and set my env to point to the new, clean minikube daemon. When I run `make test-integration`, I get this:

```console
ENG000656:service-catalog aaronschlesinger$ make test-integration
# Generate defaults
docker run --rm -ti -v /Users/aaronschlesinger/gocode/src/github.com/kubernetes-incubator/service-catalog:/go/src/github.com/kubernetes-incubator/service-catalog -v /Users/aaronschlesinger/gocode/src/github.com/kubernetes-incubator/service-catalog/.pkg:/go/pkg scbuildimage bin/defaulter-gen \
		--v 1 --logtostderr \
		--go-header-file "vendor/github.com/kubernetes/repo-infra/verify/boilerplate/boilerplate.go.txt" \
		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog" \
		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1" \
	  	--extra-peer-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog" \
		--extra-peer-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1" \
		--output-file-base "zz_generated.defaults"
Unable to find image 'scbuildimage:latest' locally
Pulling repository docker.io/library/scbuildimage
docker: Error: image library/scbuildimage not found.
See 'docker run --help'.
make: *** [.generate_files] Error 125
```

The workaround of course is to run `make clean test-integration`. I'm wondering if there's a way to `docker build` the build image if it doesn't exist, or, alternatively, push the image to a public registry.

**Comment by pmorie** [comment]
We need to add `.scBuildImage` a precondition of `test-integration` target do this, should be pretty-standard.

**Comment by duglin** [comment]
done as part of #424

**Comment by pmorie** [comment]
Closed by #424
